spec/support/shared_examples/cnab400.rb

```diff
@@ -35,7 +35,7 @@
         agencia: '1234',
         conta_corrente: '12345',
         digito_conta: '1',
-        empresa_mae: 'asd',
+        empresa_mae: 'ASD',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.